Marlin/example_configurations/delta/kossel_mini/Configuration_adv.h

```diff
@@ -1,4 +1,4 @@
-/**
+﻿/**
```

Comment:
Why do the first lines keep changing? Because of your editor.
Please save as "UTF-8" format, not "UTF-8 with BOM."